workflow-to-js-transformer/src/main/resources/Workflow.xslt

```diff
@@ -99,11 +100,14 @@
     <xsl:template match="details/settings[../typeInternalName='CompositeDocumentWorkerHandler']"><xsl:call-template name="ChainedActionTypeTemplate"/></xsl:template>
 
     <xsl:template name="ChainedActionTypeTemplate" match="details/settings[../typeInternalName='ChainedActionType']">{
-        'internal_name' : '<xsl:value-of select="../typeInternalName"/>',
-        'queueName' : '<xsl:choose><xsl:when test="queueName != ''"><xsl:value-of select="queueName"/></xsl:when><xsl:otherwise><xsl:variable name="workerNameQueueEnvValue" select="workflow_transform:TransformerFunctions.getWorkerQueueFromEnvironment(workerName)"/><xsl:choose><xsl:when test="$workerNameQueueEnvValue != ''"><xsl:value-of select="$workerNameQueueEnvValue"/></xsl:when><xsl:otherwise><xsl:value-of select="concat(workerName, 'Input')"/></xsl:otherwise></xsl:choose></xsl:otherwise></xsl:choose>',
-        'workerName' : '<xsl:value-of select="workerName"/>'<xsl:if test="customData/*">,
-        'customData' : {<xsl:apply-templates select="customData"/>}
-        </xsl:if>   }</xsl:template>
+                'internal_name' : '<xsl:value-of select="../typeInternalName"/>',
```

Comment:
Should this indenting have changed?

Reply:
Yep. I added this to try and make it line up better when actually output as JS.